recipes/dcmtk/all/conanfile.py

```diff
@@ -91,101 +82,116 @@ def requirements(self):
         if self.options.with_zlib:
             self.requires("zlib/1.2.13")
         if self.options.with_openssl:
-            if self.settings.os == "Windows":
-                # FIXME: CMake configuration fails to detect Openssl 1.1 on Windows.
-                self.requires("openssl/1.0.2u")
-            else:
-                self.requires("openssl/1.1.1t")
+            # Configuration checks fail with OpenSSL 3
+            self.requires("openssl/1.1.1u")
         if self.options.with_libpng:
             self.requires("libpng/1.6.39")
         if self.options.with_libtiff:
             self.requires("libtiff/4.4.0")
         if self.options.get_safe("with_tcpwrappers"):
             self.requires("tcp-wrappers/7.6")
 
+    @property
+    def _min_cppstd(self):
+        return 11
+
     def validate(self):
-        if hasattr(self, "settings_build") and cross_building(self) and \
-           self.settings.os == "Macos" and self.settings.arch == "armv8":
+        if not can_run(self) and self.settings.os == "Macos" and self.settings.arch == "armv8":
```

Comment:
```suggestion
        if not cross_building(self) and self.settings.os == "Macos" and self.settings.arch == "armv8":
```
I think the ``can_run()`` is not exactly the same as ``cross_building()`` and can actually return ``True`` for cross building scenarios.
(The import of ``cross_building`` would be missing)